platform/src/apis/EducationBenchmarking.Platform.Api.Establishment/Db/Edubase.cs

```diff
@@ -0,0 +1,132 @@
+using Newtonsoft.Json;
+
+namespace EducationBenchmarking.Platform.Api.Establishment.Db;
+
+public class Edubase
```

Comment:
Why is this named Edubase? Does it relate to the GIAS/edubase data, or is it going to be populated with information from the maintained / academies reports? 

I'm trying to understand the relevance of calling it Edubase here, and whether it might cause confusion with something else.

Reply:
It relates to the Edubase dataset/collection.